CHANGELOG.md

```diff
@@ -1,10 +1,120 @@
+  and data source. ([#917](https://github.com/vmware/terraform-provider-vcd/pull/917))
+* Deprecate `template_name` in favor of `vapp_template_id` in `vcd_vapp_vm` and `vcd_vm` to be able to use unique URNs instead
+  of catalog dependent names ([#931](https://github.com/vmware/terraform-provider-vcd/pull/931))
+* Deprecate `boot_image` in favor of `boot_image_id` in `vcd_vapp_vm` and `vcd_vm` to be able to use URNs instead
+  of catalog dependent names ([#931](https://github.com/vmware/terraform-provider-vcd/pull/931))
+* Deprecate `catalog_name` in favor of `vapp_template_id` or `boot_image_id`, which don't require a catalog name anymore ([#931](https://github.com/vmware/terraform-provider-vcd/pull/931))
+* Data source `vcd_nsxt_edge_cluster` deprecates `vdc` field in favor of 3 new fields to define
+  NSX-T Edge Cluster lookup scope - `vdc_id`, `vdc_group_id`, and `provider_vdc_id` ([#921](https://github.com/vmware/terraform-provider-vcd/pull/921))
+
+### NOTES
+* Add a guide and examples on [Catalog subscribing and sharing](https://registry.terraform.io/providers/vmware/vcd/latest/docs/guides/catalog_subscription_and_sharing) to the documentation ([#916](https://github.com/vmware/terraform-provider-vcd/pull/916))
+* All non-NSX-V resources and data sources use the new SDK signatures with Context and Diagnostics ([#895](https://github.com/vmware/terraform-provider-vcd/pull/895))
+* Refactor VM Creation code, which should result in identifiable parts creation for all types of
+  VMs. Behind the scenes, there are 4 different types of VMs with respective different API calls as
+  listed below ([#901](https://github.com/vmware/terraform-provider-vcd/pull/901))
+  * `vcd_vapp_vm` built from vApp template
+  * `vcd_vm` built from vApp template
+  * `vcd_vapp_vm` built without vApp template (empty VM)
+  * `vcd_vm` built without vApp template (empty VM)
+* Bump Go to 1.19 in `go.mod` as the minimum required version. ([#902](https://github.com/vmware/terraform-provider-vcd/pull/902), [#916](https://github.com/vmware/terraform-provider-vcd/pull/916))
+* Code documentation formatting is adjusted using Go 1.19 (`make fmt`) ([#902](https://github.com/vmware/terraform-provider-vcd/pull/902))
+* Adjust GitHub actions in pipeline to use the latest code ([#902](https://github.com/vmware/terraform-provider-vcd/pull/902))
+* `staticcheck` switched version naming from `2021.1.2` to `v0.3.3` in downloads section. This PR
+  also updates the code to fetch correct staticcheck. ([#902](https://github.com/vmware/terraform-provider-vcd/pull/902))
+* package `io/ioutil` is deprecated as of Go 1.16. `staticcheck` started complaining about usage of
+  deprecated packages. As a result this PR switches packages to either `io` or `os` (still the same
+  functions are used) ([#902](https://github.com/vmware/terraform-provider-vcd/pull/902))
+* Drop support for EOL VCD 10.2.x ([#903](https://github.com/vmware/terraform-provider-vcd/pull/903))
```

Comment:
Please move this bullet point to the top of the list (as it may be one of the most important for some users).

Reply:
It is already at the top of the list. Please refresh